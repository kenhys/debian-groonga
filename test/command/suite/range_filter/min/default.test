table_create Memos TABLE_NO_KEY
column_create Memos id COLUMN_SCALAR Int32
column_create Memos text COLUMN_SCALAR Text

table_create IDs TABLE_PAT_KEY Int32
column_create IDs memos_id COLUMN_INDEX Memos id

load --table Memos
[
{"id":   0, "text": "aaa"}
]

range_filter Memos id \
  --min_border include \
  --max 10 --max_border exclude
