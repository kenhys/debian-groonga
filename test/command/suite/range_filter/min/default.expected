table_create Memos TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Memos id COLUMN_SCALAR Int32
[[0,0.0,0.0],true]
column_create Memos text COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create IDs TABLE_PAT_KEY Int32
[[0,0.0,0.0],true]
column_create IDs memos_id COLUMN_INDEX Memos id
[[0,0.0,0.0],true]
load --table Memos
[
{"id":   0, "text": "aaa"}
]
[[0,0.0,0.0],1]
range_filter Memos id   --min_border include   --max 10 --max_border exclude
[[0,0.0,0.0],[[["_id","UInt32"],["id","Int32"],["text","Text"]],[1,0,"aaa"]]]
